makefile.proj

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Package" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_BaseDir>$(MsBuildThisFileDirectory)</_BaseDir>
+    <_ToolsDownloadsPath>$(_BaseDir).tools</_ToolsDownloadsPath>
+    <_ArtifactsDir>$(_BaseDir)artifacts</_ArtifactsDir>
+  </PropertyGroup>
+  <Target Name="Clean">
+    <ItemGroup>
+      <DirectoriesToClean Include="
+        $(_ToolsDownloadsPath);
+        $(_ArtifactsDir);" />
+    </ItemGroup>
+    <RemoveDir Directories="@(DirectoriesToClean)" />
+  </Target>
+
+  <Target Name="Initialize">
+    <Exec Command="dotnet restore src" />
+    <Exec Command="dotnet restore project.json --packages $(_ToolsDownloadsPath)" />
+  </Target>
+
+  <Target
+    Name="Package"
+    DependsOnTargets="CoherenceBuild;SplitPackages;DependenciesPackager">
```

Comment:
We might want to consider if we want to do $(PackageDependsOn) as its the recommended practice to do so that anyone else can plug themselves into it by doing 
```xml
<PackageDependsOn>$(PackageDependsOn);MyTarget</PackageDependsOn>
```